src/contlinear.jl

```diff
@@ -150,7 +150,7 @@ function linear1test(solver::Function, config::TestConfig)
         @test vars == [v[1], v[2]] || vars == [v[2], v[1]]
         @test MOI.canget(instance, MOI.ObjectiveFunction())
         vars = MOI.get(instance, MOI.ObjectiveFunction()).variables
-        @test vars == [v[1], v[2]] || vars == [v[2], v[1]]
+        @test vars == [v[1]]
```

Comment:
This isn't the right fix. We need a way to compare SAFs that doesn't care if there are extra zeros.

Reply:
We can use isapprox here

Reply:
I agree